.changeset/silly-jeans-wonder.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Renamed ClientSecret to AzureClientSecretCredential and ManagedIdentity to AzureManagedIdentityCredential
```

Comment:
This should probably be `minor` and then the practice is to prefix this with `**BREAKING**:`

Here's an example for reference: https://github.com/backstage/backstage/blob/6eb90068a88272601bdc030604bf436b21ae31f5/.changeset/silver-cars-type.md?plain=1#LL7C1-L7C14

Reply:
No worries, this isn't breaking as these were all added in #17780, which hasn't been released yet. It's why I was eager to ship this before next Tuesday and asked for the separate PR. Thank you for keeping an eye out for this though! 🙏